cmd/server/app/webhook_update.go

```diff
@@ -217,3 +217,52 @@ func updateGithubRepoHooks(
 
 	return nil
 }
+
+func wireUpProviderManager(cfg *serverconfig.Config, store db.Store) (manager.ProviderManager, error) {
+	cryptoEng, err := crypto.EngineFromAuthConfig(&cfg.Auth)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create crypto engine: %w", err)
+	}
+	fallbackTokenClient := ghprovider.NewFallbackTokenClient(cfg.Provider)
+	providerStore := providers.NewProviderStore(store)
+	githubProviderManager := ghmanager.NewGitHubProviderClassManager(
+		&ratecache.NoopRestClientCache{},
+		clients.NewGitHubClientFactory(telemetry.NewNoopMetrics()),
+		&cfg.Provider,
+		fallbackTokenClient,
+		cryptoEng,
+		nil, // whManager not needed here (only when creating/delete webhooks)
+		store,
+		nil, // ghProviderService not needed here
+	)
+
+	return manager.NewProviderManager(providerStore, githubProviderManager)
+}
+
+func wireUpDB(ctx context.Context, cfg *serverconfig.Config) (db.Store, error) {
```

Comment:
Some of these were factored out to reduce cyclomatic complexity of the `runCmdWebhookUpdate`